apps/ledger-live-mobile/src/images/devices/NanoX.tsx

```diff
@@ -1,8 +1,14 @@
 import React from "react";
 import Svg, { Path } from "react-native-svg";
 
-const NanoX = ({ color }: { color: string }) => (
-  <Svg width={85} height={87} fill="none">
+const NanoX = ({
+  size = 85,
+  color = "neutral.c100",
+}: {
+  size?: number;
+  color?: string;
+}) => (
+  <Svg width={size} height={size} fill="none" viewBox="0 0 85 87">
```

Comment:
those SVG will be used again in the onboarding-device-selection in https://github.com/LedgerHQ/ledger-live/pull/1486, we should double check that it does not change anything @thomasrogerlux

Reply:
The props that were added are optional and have a default value, it shouldn't break anything.